tests/Fixtures/Integration/set/@PHPUnit56Migration-risky.test-out.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class FooTest extends \PHPUnit\Framework\TestCase {
+    public function test_dedicate_assert($foo) {
+        $this->assertNull($foo);
+        $this->assertInternalType('array', $foo);
+        $this->assertNan($foo);
+        $this->assertIsReadable($foo);
+    }
+
+    /**
+     * Foo.
+     */
+    function test_php_unit_no_expectation_annotation_32()
+    {
+        $this->expectException(\FooException::class);
+        $this->expectExceptionMessage('');
```

Comment:
this has to be dropped after discussion with Sebastian and change in 5.7.x and 6.x line